### PR TITLE
Fixed incorrect written display of star rating

### LINKED ISSUE #3058: Display of star rating is incorrect
**Actual Behaviour**

The display of number of stars is different from number of stars mentioned label. 

**Expected Behaviour**

The display of number of stars should be same with number of stars mentioned in label.

**Steps to reproduce it**

Hover on stars of rating on any skill.

**Screenshots of the issue**

Screenshot (95)

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by simranaggarwal1999**
Fixes #3058 

Changes: Fixed incorrect written display of star rating

Description: The written display of number of stars is now same with number of stars mentioned in label.

Demo Link: https://pr-3414-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 

Before change
image

After change
image

starrating

### CODE REVIEW

**Comment by anishagg17** [comment]
Screenshot_20200203-010306~2

You have changed the total number of reviews too. It was not supposed

File: src/components/cms/SkillCardScrollList/SkillCard.js [hunk 1]
```diff
@@ -135,8 +135,8 @@ class SkillCard extends Component {
           examples,
           image,
           staffPick = false;
-        let averageRating = 0,
-          totalRating = 0;
+        let averageRating = 0;
+        // totalRating = 0;
```
Comment (mohitm15) [inline src/components/cms/SkillCardScrollList/SkillCard.js]: You can remove the commit.

**Comment by simranaggarwal1999** [comment]
@anishagg17 But I just displayed the average rating of reviews which was supposed to be done.

**Comment by mohitm15** [comment]
@simranaggarwal1999 Please squash your commits.

**Comment by simranaggarwal1999** [comment]
I am unable to sqaush commits. I tried really hard.

**Comment by snitin315** [comment]
> I am unable to sqaush commits. I tried really hard.

to squash last N commits
``` 
git rebase -i HEAD~N
then repalce pick with squash for the commits you want to squash.
save

**Comment by mohitm15** [comment]
> I am unable to sqaush commits. I tried really hard.

https://www.youtube.com/watch?v=viY1BbKZhSI This helps you

**Comment by anishagg17** [comment]
> @anishagg17 But I just displayed the average rating of reviews which was supposed to be done.

You have made some extra changes too. which were not supposed

**Comment by Orbiter** [comment]
@mohitm15 @anishagg17 what is the status of this fix? It looks like it removes a bad inconsistency but has this reached a final state?

**Comment by akshatnitd** [comment]
Closing due to inactivity.

**Pull request closed without merging**